plugin/vim-copilot-ignore.vim

```diff
@@ -56,5 +50,5 @@ endfunction
 " Automatically run the function when entering a buffer
 augroup copilotignore
   autocmd!
-  autocmd BufEnter * call s:CheckAndDisableCopilot()
+  autocmd BufNew,BufRead * call s:CheckAndDisableCopilot()
```

Comment:
Seems this is a better approach since if you re-enable Copilot for that buffer, the old version would disable it again whenever you switched to that buffer.